src/main/java/no/fdk/dataservicecatalog/controller/CatalogHandler.java

```diff
@@ -23,7 +24,7 @@
         return dcatApNoModelService
                 .buildCatalogsModel()
                 .doOnSuccess(model -> log.info("Successfully built catalogs model"))
-                .doOnError(error -> log.error("Failed to build catalogs model", error))
+                .doOnError(error -> log.error("{}: Failed to build catalogs model", ExceptionUtils.getStackTrace(error)))
```

Comment:
Ikke gjør dette her. Bruk logback config. For eks.

```
<encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
  <providers>
    <pattern>
      <!-- the pattern that defines what to include -->
      <pattern>
        { "level": "%level" , "message" : "%exception: %logger | %message " }}
      </pattern>
    </pattern>
  </providers>
</encoder>
```

Reply:
Ah, slå de sammen vha patter, mye bedre :+1: 
Skal slenge ut en fiks på samtlige så langt

Reply:
Har du tilfeldigvis peiling på om det er mulig å få tak i `artifactId` fra pom i logback-config?